code-challenges.test.js

```diff
@@ -33,19 +33,96 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe('divideByThree', () => {
+    it('takes an object as an argument and decides if the number inside it is evenly divisible by three or not.', () => {
+      const object1 = { number: 15 }
+      // Expected output: "15 is divisible by three"
+      const object2 = { number: 0 }
+      // Expected output: "0 is divisible by three"
+      const object3 = { number: -7 }
+      // Expected output: "-7 is not divisible by three"
+      expect(divideByThree(object1.number)).toEqual("15 is divisible by three")
+      expect(divideByThree(object2.number)).toEqual("0 is divisible by three")
+      expect(divideByThree(object3.number)).toEqual("-7 is not divisible by three")
```

Comment:
Make sure to pay attention to the prompt, your function should take an object as an argument. In your test, you are giving the function a number as an argument.